### PR TITLE
add redundancy segment checker

### PR DESCRIPTION
**Pull request by weiliu1031**
Signed-off-by: wei.liu <wei.liu@zilliz.com>
Related to #19526

### CODE REVIEW

**Comment by weiliu1031** [comment]
rerun ut

**Comment by weiliu1031** [comment]
rerun ut

File: internal/querynode/data_sync_service_test.go [hunk 1]
```diff
@@ -198,7 +198,7 @@ func (s *DataSyncServiceSuite) TestRemoveEmptyFlowgraphByChannel() {
 		channelName := fmt.Sprintf("%s_%d_1", Params.CommonCfg.RootCoordDml, defaultCollectionID)
 		deltaChannelName, err := funcutil.ConvertChannelName(channelName, Params.CommonCfg.RootCoordDml, Params.CommonCfg.RootCoordDelta)
 		s.Require().NoError(err)
-		err = s.dsService.metaReplica.addSegment(defaultSegmentID, defaultPartitionID, defaultCollectionID, channelName, defaultSegmentVersion, segmentTypeSealed)
+		err = s.dsService.metaReplica.addSegment(defaultSegmentID, defaultPartitionID, defaultCollectionID, channelName, defaultSegmentVersion, defaultSegmentStartPosition, segmentTypeSealed)
```
Comment (jiaoew1991) [inline internal/querynode/data_sync_service_test.go:201]: too many params in `addSegment`, what about refactor it to `addSegment(opt *Opt)`

File: internal/util/paramtable/component_param.go [hunk 2]
```diff
@@ -813,6 +815,15 @@ func (p *queryCoordConfig) initCheckInterval() {
 	p.CheckInterval = time.Duration(checkInterval) * time.Millisecond
 }
 
+func (p *queryCoordConfig) initCheckRedundancyInterval() {
+	interval := p.Base.LoadWithDefault("queryCoord.checkRedundancyInterval", "60000")
+	checkInterval, err := strconv.ParseInt(interval, 10, 64)
+	if err != nil {
+		panic(err)
```
Comment (congqixia) [inline internal/util/paramtable/component_param.go:822]: Panic behavior is OK. But the message is vague and cannot explain why querycoord panics to users.

**Pull request closed without merging**